### ISSUE TITLE
PrismicLink issue for new tab links

### ISSUE THREAD

**Issue by derhyl** [nuxt-modules/prismic#181]
### Versions

- nuxt: 3.0.0
- @nuxtjs/prismic: ^3.0.0-rc.2
- node: 18

### Steps to reproduce

Create a PrismicLink with target blank. 

### What is Expected?

Link should open in new tab.

### What is actually happening?

I receive the correct data from Primic : 
```
{
"link_type": "Web",
"url": "https://...",
"target": "_blank"
}
```
BUT link opens on same tab when link is provided to PrismicLink

**Comment by lihbr** [comment]
Hey there, thanks for opening an issue!

I was able to reproduce: https://stackblitz.com/edit/github-kq4ync?file=app.vue

I'll fix that soon, in the meantime you can quickfix it by providing in your Prismic options:

```diff
  prismic: {
+   components: {
+     linkExternalComponent: "a"
+   }
  }
```

**Comment by lihbr** [comment]
Hey there, this should be fixed in `@prismicio/vue@3.1.3`, can you try updating it with `npm update @prismicio/vue` (or similar)?

**Comment by lihbr** [comment]
Closing, feel free to reopen if still experiencing
